erpnext/manufacturing/doctype/production_plan/production_plan.js

```diff
@@ -60,7 +60,12 @@ frappe.ui.form.on('Production Plan', {
 		if (frm.doc.docstatus === 1 && frm.doc.po_items
 			&& frm.doc.status != 'Completed') {
 			frm.add_custom_button(__("Work Order"), ()=> {
-				frm.trigger("make_work_order");
+				if (frm.doc.status==="Material Received") {
```

Comment:
@hitenshKharva if the Material Request table is empty then it should be allowed to create a work order.  following change irrespective of the material requested or not it still gives validation error for work order

Reply:
@vishdha Yes. Will fix it.